thrift/src/main/thrift/atoms/storyquestions.thrift

```diff
@@ -22,10 +24,12 @@ struct QuestionSet {
   3: required list<Question> questions
 }
 
+
 struct StoryQuestionsAtom {
   1: required string relatedStoryId
-  2: required RelatedStoryLinkType relatedStoryLinkType 
+  2: required RelatedStoryLinkType relatedStoryLinkType
   3: required string title
   4: optional list<QuestionSet> editorialQuestions
   5: optional list<QuestionSet> userQuestions
-}
\ No newline at end of file
+  6: optional list<shared.NotificationProvider> providers
```

Comment:
Ah yes, that's because initially I had defined a `union Provider` but then I remembered they were a headache to work with. Changing that.